iOSTransport/ALXGoogleMapsViewController.m

```diff
@@ -0,0 +1,138 @@
+//
+//  ALXGoogleMapsViewController.m
+//  iOSTransport
+//
+//  Created by Alex De Souza Campelo Lima on 3/7/15.
+//  Copyright (c) 2015 Alex De Souza Campelo Lima. All rights reserved.
+//
+
+#import "ALXGoogleMapsViewController.h"
+
+@interface ALXGoogleMapsViewController ()
+
+@property (nonatomic) GMSMapView *mapView;
+@property (nonatomic) GMSMarker *marker;
+@property (nonatomic) GMSGeocoder *geocoder;
+@property (nonatomic) NSString *route;
+@property (nonatomic) BOOL infoProcessed;
+
+@end
+
+@implementation ALXGoogleMapsViewController
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    
+    //Set camera on Florianopolis
+    GMSCameraPosition *camera = [GMSCameraPosition cameraWithLatitude:-27.590968
```

Comment:
The magic numbers problem were solved on commit c47eed3c51025c6269663774849ed6c68cc14879. The solution is the same as mentioned before.